calico-cloud/image-assurance/scan-image-registries.mdx

```diff
@@ -113,27 +113,52 @@ You can scan images and report results back to {{prodname}}, or scan images loca
 - `--warn_threshold` - CVSS threshold for Warn scan results. Range from 0.0 - 10.0.
 - `--fail_threshold` - CVSS threshold for Fail scan results. Range from 0.0 - 10.0.
 - `--vulnerability_db_path` - path to a folder to store vulnerability data (defaults to `$XDG_CACHE_HOME`; if it is not set, defaults to `$HOME/.cache`).
+- `--input_file` - Input file containing image URLs.
```

Comment:
--input_file and --output_file require arguments, no? Something like this might be clearer:

`input_file <file-path>` Path to input file containing image URLs.

Reply:
Also: What kind of file? What format?  (Line 290 says JSON works ...)